### PR TITLE
intro to CDC in Aura and redirect to full docs

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-350.surge.sh

Updated pages:
- [Connecting applications](https://neo4j-docs-aura-350.surge.sh/aura/auradb/connecting-applications/overview)

File: modules/ROOT/pages/auradb/connecting-applications/overview.adoc [hunk 1]
```diff
@@ -7,4 +7,10 @@ You can use the official link:{neo4j-docs-base-uri}/create-applications/[drivers
 Regardless of what language you use, you will need to provide the following information to connect to an AuraDB instance:
 
 * `uri` - The *Connection URI* for your AuraDB instance. You can copy this from the instance card or details page in the Console.
-* `username` and `password` - The *Username* and *Password* for your AuraDB instance. You can copy or download these during the instance creation process.
\ No newline at end of file
+* `username` and `password` - The *Username* and *Password* for your AuraDB instance. You can copy or download these during the instance creation process.
+
+== Change data capture
+
+Change Data Capture (CDC) allows you to capture and track changes to your database in real-time, enabling you to keep your other data sources up to date with Neo4j. With CDC, you can identify and respond to changes (create, update, and delete) on nodes and relationships as they happen, and integrate these changes into other systems and applications.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/auradb/connecting-applications/overview.adoc:14]:
```suggestion
Change Data Capture (CDC) allows you to capture and track changes to your database in real time, enabling you to keep your other data sources up to date with Neo4j. 
With CDC, you can identify and respond to changes (create, update, and delete) on nodes and relationships as they happen, and integrate these changes into other systems and applications.
```

File: modules/ROOT/pages/auradb/connecting-applications/overview.adoc [hunk 2]
```diff
@@ -7,4 +7,10 @@ You can use the official link:{neo4j-docs-base-uri}/create-applications/[drivers
 Regardless of what language you use, you will need to provide the following information to connect to an AuraDB instance:
 
 * `uri` - The *Connection URI* for your AuraDB instance. You can copy this from the instance card or details page in the Console.
-* `username` and `password` - The *Username* and *Password* for your AuraDB instance. You can copy or download these during the instance creation process.
\ No newline at end of file
+* `username` and `password` - The *Username* and *Password* for your AuraDB instance. You can copy or download these during the instance creation process.
+
+== Change data capture
```
Comment (AlexicaWright) [inline modules/ROOT/pages/auradb/connecting-applications/overview.adoc:12]:
```suggestion
== Change Data Capture
```

**Comment by fiquick** [comment]
[CDC aura intro linking to full docs](https://trello.com/c/aWU3KLph/318-cdc-aura-intro-linking-to-full-docs)

File: modules/ROOT/pages/auradb/connecting-applications/overview.adoc [hunk 3]
```diff
@@ -7,4 +7,10 @@ You can use the official link:{neo4j-docs-base-uri}/create-applications/[drivers
...
+
+Change Data Capture (CDC) allows you to capture and track changes to your database in real-time, enabling you to keep your other data sources up to date with Neo4j. With CDC, you can identify and respond to changes (create, update, and delete) on nodes and relationships as they happen, and integrate these changes into other systems and applications.
+
+See link:https://neo4j.com/docs/cdc/enablecdc/aura for more information about setting up CDC, configuring it to capture the changes, and querying those changes for further processing, such as replicating to another system.
```
Comment (recrwplay) [inline modules/ROOT/pages/auradb/connecting-applications/overview.adoc:16]:
```suggestion
See link:{neo4j-docs-base-uri}/cdc/enablecdc/aura for more information about setting up CDC, configuring it to capture the changes, and querying those changes for further processing, such as replicating to another system.
```

For links to pages in other docs we can use the `{neo4j-docs-base-uri}` attribute, which is defined in the playbook

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-350.surge.sh

Updated pages:
- [Connecting applications](https://neo4j-docs-aura-350.surge.sh/aura/auradb/connecting-applications/overview)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-350.surge.sh

Updated pages:
- [Connecting applications](https://neo4j-docs-aura-350.surge.sh/aura/auradb/connecting-applications/overview)

**Pull request merged**